### DISCUSSION TITLE
Linux on Intel 4004

### DISCUSSION THREAD

**Discussion by Vutshi** [jbruchon/elks#2033] [General]
Apparently, 8088 is not the limit for Linux :)

Youtube:
[![IMAGE ALT TEXT](http://img.youtube.com/vi/IPGLStPP9p8/0.jpg)](http://www.youtube.com/watch?v=IPGLStPP9p8 "Video Title")

https://dmitry.gr/?r=05.Projects&proj=35.+Linux4004

**Comment by ghaerr** [comment]
Wow, what an article! Did I get that right, first he writes a 4004 emulator, then a MIPS emulator written for 4004, then runs a MIPS Linux on the MIPS emulator on the 4004 emulator, then on real 4004 hardware. Amazing, and an ingenious plan with a lot of work :)

**Reply by Vutshi** [reply]
While his plan is absolutely brilliant, I'm even more struck (or frustrated) by the effort it took to make the video. On one hand, we have incredible level of modern hardware engineering resulting in the iPhone, capable of taking and storing 379GB of photos. On the other, it's absurd that modern level of software engineering makes it almost impossible to simply extract them.
